spa/src/pages/Connections/GHEnterpriseConnections/GHEnterpriseApplication.tsx

```diff
@@ -0,0 +1,64 @@
+import { DynamicTableStateless } from "@atlaskit/dynamic-table";
+import { useState } from "react";
+import styled from "@emotion/styled";
+import Heading from "@atlaskit/heading";
+import ChevronRightIcon from "@atlaskit/icon/glyph/chevron-right";
+import ChevronDownIcon from "@atlaskit/icon/glyph/chevron-down";
+import { head, getGHSubscriptionsRows } from "../../../utils/dynamicTableHelper";
+import { GitHubEnterpriseApplication } from "../../../rest-interfaces";
+
+const Wrapper = styled.div`
+	display: flex;
+	align-items: center;
+	flex-direction: column;
+`;
+
+const ApplicationHeader = styled.div`
```

Comment:
We shouldn't be using styled components in this way, according to the [Coding Conventions doc](https://hello.atlassian.net/wiki/spaces/OTFS/pages/1424951560/Arc+Coding+Conventions). Check out the `Emotion CSS Naming conventions + styled vs css prop` line here. We should be using `css props` instead.

```Component.styles.tsx

export const containerStyles = css`
    display: flex;
	align-items: center;
	justify-content: center;
	flex-direction: column;
`

export const contentStyles = css`
	font-weight: bold;
`;
```

```
Component.tsx

<div css={containerStyles}>
	<p css={contentStyles}>I'm a component!</p>
</div>
```

Reply:
It's important that since we are writing this code from scratch that we try to follow these conventions to set a good base to work off of :)

Reply:
It is a big change and will overpopulate this PR, so will be done in a [separate issue](https://softwareteams.atlassian.net/browse/ARC-2467).

Reply:
Done